conceptnet5/vectors/evaluation/wordsim.py

```diff
@@ -440,8 +518,43 @@ def evaluate_raw(frame, subset='dev'):
     results.loc['ws353'] = ws_score
     results.loc['ws353-es'] = ws_es_score
     results.loc['ws353-ro'] = ws_ro_score
-    results.loc['semeval17-2a'] = semeval_monolingual
-    results.loc['semeval17-2b'] = semeval_crosslingual
+
+    if semeval_scope == 'global':
+        semeval_monolingual = evaluate_semeval_monolingual_global(frame)
+        semeval_crosslingual = evaluate_semeval_crosslingual_global(frame)
+        results.loc['semeval17-2a'] = semeval_monolingual
+        results.loc['semeval17-2b'] = semeval_crosslingual
+    else:
+        semeval_en = evaluate_semeval_monolingual(frame, 'en')
+        semeval_de = evaluate_semeval_monolingual(frame, 'de')
+        semeval_es = evaluate_semeval_monolingual(frame, 'es')
+        semeval_it = evaluate_semeval_monolingual(frame, 'it')
+        semeval_fa = evaluate_semeval_monolingual(frame, 'fa')
+        semeval_en_de = evaluate_semeval_crosslingual(frame, 'en', 'de')
```

Comment:
(drive-by: if you want to be fancy, https://docs.python.org/3/library/itertools.html#itertools.combinations)